zendesk_rules/zendesk_new_api_token.py

```diff
@@ -0,0 +1,16 @@
+def rule(event):
+    return event.get("source_type") == "api_token" and event.get("action", "") in {
+        "create",
+        "destroy",
+    }
```

Comment:
nit if we define
```python
{
        "create",
        "destroy",
    }
```
once, we don't have to re-define this set  for  each event processed.